src/npm-dependency-version-checker.js

```diff
@@ -12,7 +13,7 @@ class NPMDependencyVersionChecker extends DependencyVersionChecker {
     let jsonPath;
     try {
       jsonPath = resolve.sync(this.name + '/package.json', {
-        basedir: addon.root,
+        basedir: addon.root || getProject(addon).root,
```

Comment:
I'm not certain this is the correct behavior, do you have a specific reproduction scenario?

Reply:
I've described a scenario in https://github.com/ember-cli/ember-cli-version-checker/issues/70 😊